### PR TITLE
feat: first version of about

### PR DESCRIPTION
**Pull request by Nounoursdestavernes**

### CODE REVIEW

File: pages/about.vue [hunk 1]
```diff
@@ -3,56 +3,44 @@
...
-				>
-					<a href="https://twitter.com/tsgctf" target="_blank" rel="noopener noreferrer">Tweets from @tsgctf</a>
-				</timeline>
-			</li>
 		</ul>
 		<h3>Sponsor</h3>
-		<a href="https://flatt.tech/" target="_blank" rel="noopener noreferrer">
-			<img src="https://ctf.tsg.ne.jp/images/flatt_image.png" :style="{maxWidth: '60rem'}">
+		<a href="https://www.infomaniak.com/" target="_blank" rel="noopener noreferrer">
+			<img src="./infomaniak.png">
```
Comment (yyewolf) [inline pages/about.vue]: Has this been tested ?

File: pages/about.vue [hunk 2]
```diff
@@ -3,56 +3,44 @@
...
+		<p>Do not hesitate to take a look to their offers ! <a href="https://www.infomaniak.com/">https://www.infomaniak.com/</a> </p>
 		<h3>Prizes</h3>
-		<ul>
-			<li>1st: 1,000USD</li>
-			<li>2nd: 500 USD</li>
-			<li>3rd: 200 USD</li>
-		</ul>
+
+		<p>TBD</p>
+
```
Comment (yyewolf) [inline pages/about.vue]: Include the prizes for ik, afawk these are the only prizes we have

File: pages/about.vue [hunk 3]
```diff
@@ -77,24 +65,11 @@ export default {
...
-				{ url: 'https://twitter.com/satos___jp', name: '@satos' },
-				{ url: 'https://twitter.com/sitositositoo', name: 'settyan117' },
-				{ url: 'https://twitter.com/shell_mug', name: '@shell_mug' },
-				{ url: 'https://twitter.com/syobon_hinata', name: '@syobon_hinata (fabon)' },
-				{ url: 'https://twitter.com/shion_urei', name: '@shion_urei' },
+				{ url: 'https://yewolf.fr', name: 'Yewolf' },
+				{ url: 'https://github.com/Nounoursdestavernes', name: 'Nounours des tavernes' },
+				{ url: 'https://example.com/', name: 'Nathan' },
+				{ url: 'https://example.com/', name: 'Thib' }, 
+				{ url: 'https://example.com/', name: 'Mada07' },
```
Comment (yyewolf) [inline pages/about.vue]: Missing github links

Comment (Nounoursdestavernes) [reply pages/about.vue]: done

File: pages/about.vue (hunk 1 above)
Comment (Nounoursdestavernes) [reply pages/about.vue]: done

File: pages/about.vue (hunk 2 above)
Comment (Nounoursdestavernes) [reply pages/about.vue]: done

**Pull request merged**